course/models.py

```diff
@@ -114,6 +123,13 @@ class Meta:
         verbose_name_plural = 'Курсы'
 
     def clean(self):
+        image_format = self.image.name.split('.')[-1].lower()
+        if image_format not in settings.ALLOWED_IMAGE_FORMATS:
+            raise ValidationError('Пожалуйста загружите фотографию в формате: jpg, jpeg или png!')
+
+        if self.image.width / 960 != self.image.height / 540:
```

Comment:
This probably looks better: 
`if self.image.width / self.image.height != 16/9`

Moreover, putting `16/9` to constants is preferred. For instance:

```
from django.conf import settings
if self.image.width / self.image.height != settings.COURSE_PHOTO_RATIO
```

Reply:
Wow! Cool!